Cargo.toml

```diff
@@ -59,22 +59,19 @@ lm75 = "0.2"
 enum-iterator = "1.4.1"
 rand_xorshift = "0.3.0"
 rand_core = "0.6.4"
-minimq = { git = "https://github.com/quartiq/minimq" } # "0.8"
+minimq = "0.8.0"
+# patch with https://github.com/rust-embedded-community/usb-device/pull/129
 usb-device = "0.2.9"
 usbd-serial = "0.1.1"
 # Keep this synced with the miniconf version in py/setup.py
-miniconf = { git = "https://github.com/quartiq/miniconf.git" } # "0.9"
+miniconf = "0.9.0"
 smoltcp-nal = { version = "0.4.1", features = ["shared-stack"]}
 bbqueue = "0.5"
 
 [dependencies.stm32h7xx-hal]
 version = "0.15.0"
 features = ["stm32h743v", "rt", "ethernet", "xspi", "usb_hs"]
 
-[patch.crates-io.usb-device]
-git = "https://github.com/ryan-summers/usb-device"
```

Comment:
Yes. Let's release. Since the issue is in a upstream crate presumable fixable by a patch release, and only affects windows, I'm happy to follow up on that with a patch here as well later.